### PR TITLE
feat(spdx2): Ignore files with no info in SPDX reports

### LINKED ISSUE #1161: Ignore files with no info in SPDX reports
### Description

When Fossology generates SPDX reports, it includes an entry for each file (at least 8 lines + 2 empty lines).
When dealing with large projects, this leads to huge SPDX reports (>200MB in my case), which are too large to be processed by existing SPDX tools.

However, many files avec no relevent information (no asserted licence, no copyright, no comment, ...) and could be safely excluded from these reports.

For example, I have no use for such an entry:
```
##File

FileName: foo/bar.png
SPDXID: SPDXRef-item462
FileChecksum: SHA1: 0678ec334dd25118f83e099fca0502b9eb9c2863
FileChecksum: MD5: 0cb846a3fed64741f62e67ca6b6ea157
LicenseConcluded: NOASSERTION
LicenseInfoInFile: NOASSERTION
FileCopyrightText: NOASSERTION
```

#### Steps followed and expected result

When generating an SPDX report, the user would be able to select a "Light-SPDX" report, in which  entries for which no meaningful information are not included.

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
## Description

Implement ignore files with no info in SPDX reports (see #1161)

## How to test

1) Activate "ignore files with no info in SPDX" in info page and launch an export SPDX. 
2) Verify only files with info are in SPDX export.

closes #1161 

refofNewReportInfoPage

### CODE REVIEW

**Comment by ag-pom** [comment]
as discussed in PR #1349 
Need a separate page for displaying report info,
a new field in report_info table to store spdx specific values,

**Comment by ag4ums** [comment]
as discussed in PR #1349
Need a separate page for displaying report info,
a new field in report_info table to store spdx report specific values,

**Comment by ag4ums** [comment]
@shaheemazmalmmd please fix the CI failing due to code alignment

**Comment by shaheemazmalmmd** [comment]
@ag4ums : test cases corrected, Please check now.

**Comment by ag4ums** [comment]
Found that files with all `NOASSERTION` result are still appearing in the report.

File: src/www/ui/admin-upload-edit.php [hunk 1]
```diff
@@ -118,7 +117,6 @@ public function Output()
       return "<h2>$text</h2>";
     }
 
-    $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc);
```
Comment (ag4ums) [inline src/www/ui/admin-upload-edit.php]: seems this line is removed  : $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc); 
buy why?

**Comment by shaheemazmalmmd** [comment]
@ag4ums : fixed the issue with `NOASSERTION` please check

**Pull request merged**